app/mailers/shf_mailer.rb

```diff
@@ -1,2 +1,68 @@
-class ShfMailer < ApplicationMailer
+class ShfMailer < Devise::Mailer
+
+  # @see https://github.com/mailgun/mailgun-ruby/docs for examples
+  # @see https://github.com/plataformatec/devise/wiki/How-To:-Use-custom-mailer
+  #   for instructions on using this custom mailer with Devise
+
+  default from: ENV['SHF_NOREPLY_EMAIL']
+  layout 'mailer'
+
+  DEFAULT_MAILGUN_DOMAIN = 'sverigeshundforetagare.se'
```

Comment:
Ya, I was overthinking this.  I was trying to provide a backup in case no ENV was defined.  But if no ENV is defined, then there _should_ be an error.